docs/en/concepts-and-designs/lal.md

```diff
@@ -240,6 +272,69 @@ metrics:
     exp: http_response_time.sum(['le', 'service', 'instance']).increase('PT5M').histogram().histogram_percentile([50,70,90,99])
 ```
 
+#### SlowSql
```

Comment:
```suggestion
- SlowSql
```

I think `metrics` uses this format?

Reply:
But method in SlowSQL use this format too . it will can not distinguish.

Reply:
You could read the preview page of markdown, https://github.com/apache/skywalking/blob/8d4bafc92fd8adda5c61eaf0a1970b127c0693ca/docs/en/concepts-and-designs/lal.md

<img width="279" alt="image" src="https://user-images.githubusercontent.com/5441976/189074830-18947c4e-3882-430e-9131-c93a3f48ea93.png">

This is totally different from others.